completion_aggregator/management/commands/run_aggregator_service.py

```diff
@@ -0,0 +1,49 @@
+"""
+run_aggregator_service management command.
+
+Usage:
+
+    ./manage.py run_aggregator_service
+
+Performs the actual aggregation.
+
+For continuous aggregation, set a cron job to run this task periodically.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+
+from django.core.management.base import BaseCommand
+
+from ...aggregator import perform_aggregation
+
+
+class Command(BaseCommand):
+    """
+    run_aggregator_service management command.
+    """
+
+    def handle(self, *args, **options):
+        """
+        Run the aggregator service.
+        """
+        self.set_logging(options['verbosity'])
+        perform_aggregation()
```

Comment:
Need to be able to run `perform_cleanup()` from a management command as well, correct?  Maybe just add an option here to run cleanup instead of aggregation?

Reply:
I think for now, I'll just add it at the end of an aggregation run.  We can revisit later if there's a valuable usecase for deferring cleanup.

Reply:
It was just as easy to create a cleanup task.